src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Resources/views/translation.html.php

```diff
@@ -1,7 +1,7 @@
 This template is used for translation message extraction tests
 <?php echo $view['translator']->trans('single-quoted key') ?>
 <?php echo $view['translator']->trans('double-quoted key') ?>
-<?php echo $view['translator']->trans(<<<EOF
+<?php echo $view['translator']->trans(<<<'EOF'
```

Comment:
this must be reverted. It is precisely testing the extraction from heredoc (extracting from nowdoc is tested 3 lines later)